packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -179,6 +179,30 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 				const globalBlockInspectorAnimationSettings =
 					select( blockEditorStore ).getSettings()
 						.blockInspectorAnimation;
+
+				// Get the name of the block that will allow it's children to be animated.
+				const animationParent =
```

Comment:
Related to my other comments about the API, if we leave this here, maybe it's better to extract this to a separate hook. What do you think?